### ISSUE TITLE
row(entry(content, attrs)) not working as expected

### ISSUE THREAD

**Issue by welguisz** [ludocracy/Ruby-Dita#10]
Gem Version:  0.5.1

`entry('text', namest: 0, nameend: 5)`produces  

`<entry namest="0" nameend="5">text</entry>`

which is correct

When I do the following `row(entry('text', namest: 0, nameend: 5))`, I expect the following:

`<row><entry namest="0" nameend="5">text</entry></row>`

Instead I get the following:

`<row><entry>text</entry></row>`

**Comment by welguisz** [comment]
I tried the following and it seems to work:

`row([entry('text', namest: 0,  nameend: 5)])`

This produces:

`<row><entry namest="0" nameend=5">text</entry><row>`

**Comment by ludocracy** [comment]
yes, row expects an array of entries, and if in doubt, the attributes
should be made explicitly into a hash e.g. {namest: 0, namend: 5}
